### DISCUSSION TITLE
Error starting server v0.2.8 - windows

### DISCUSSION THREAD

**Discussion by erickson1999** [pocketbase/pocketbase#141] [Q&A] [answered]
Hi all, I am having the following problem starting my pocketbase server: "listen tcp: lookup localhost on 200.48.225.130:53 no such host".

**Comment by ganigeorgiev** [comment, marked as answer]
Several users reported similar issues on Windows (eg. https://github.com/pocketbase/pocketbase/issues/106, https://github.com/pocketbase/pocketbase/issues/52).
You could try starting the server with `--http="127.0.0.1:8090"`, eg.
```sh
pocketbase.exe serve --http="127.0.0.1:8090"
```
_(or specify the public address and port of your server)_

I'm not sure what is actually the reason for this and why often users don't have the localhost loopback ip address enabled by default.
On my Windows 10 VM PocketBase starts just fine without further configuration to be necessary.

**Reply by erickson1999** [reply]
thank you that solved my problem
